core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -817,6 +822,11 @@ public String responseContentPreview() {
         return responseContentPreview;
     }
 
+    @Override
+    public void responseContentPreview(@Nullable String responseContentPreview) {
+        this.responseContentPreview = responseContentPreview;
```

Comment:
- `requireNonNull`
- Should not allow setting after reaching `RESPONSE_CONTENT` availability, because otherwise the preview becomes mutable.